### DISCUSSION TITLE
Presa ZigBee tipo C (italiana).

### DISCUSSION THREAD

**Discussion by nicktheone** [indomus/forum#2726] [GENERALE - Discussioni generiche]
Mi chiedevo se esistesse una presa del genere sul mercato perché tutto quello che ho trovato sono prese tipo F (schuko). Volendo connettere un dispositivo col pin di messa a terra sulla spina dovrei altrimenti usare un adattatore a monte e uno a valle della presa smart.

**Comment by M0ntyC0r4** [comment]
ecco, allora...
mi dispiace non aver visto prima sto post. io qualcosa mi pare avessi trovato, perché stavo cercando una multipresa (non ciabatta, per questioni di spazio) che avesse sia la presa elettrica che almeno una presa usb-a, comandabili separatamente. nella mia ricerca, fra le varie, avevo trovato qualcosa che non fosse con spina siemens (posto che alla siemens, se ha tre buchi e non due, puoi mettere anche una spina normale di solito).

Ora, il problema è che anche ammesso che ritrovi il link, è su amazon e mi hanno detto che non posso postare link commerciali (calcola che ormai ogni sito che vende sta roba ha il suo store, quindi un po' tutto è commerciale).

**Comment by M0ntyC0r4** [comment]
**_IMMAGINE RIMOSSA PER RIFERIMENTO A STORE_**

**Reply by snakuzzo** [reply]
Se vuoi postare, posta solo l'immagine.
Grazie.

**Reply by M0ntyC0r4** [reply]
Guarda, ho cercato su Google, mi sono ricordato che questo tizio chiedeva spine no siemens, ho fatto lo screenshot e l'ho postato, per fargli un favore.

Potevi lasciarle finché non aveva letto. 

Se cancelli, non riposto proprio nulla. Cazzi suoi!! Sai che me ne frega a me?!? Quando scrivo io non mi risponde quasi mai nessuno...ringraziasse il cielo che avevo risposto.
